### PR TITLE
Show current player

### PR DESCRIPTION
**Pull request by Jessiewithani**
Hey everyone,

This branch show's the current player on the display, increments/decrements score on the DOM, hides the question after the player submits their answer and keeps track when the round has ended.

@robbiejaeger

### CODE REVIEW

**Comment by SamanthaLFreeman** [comment]
Looks great, thank you!!

File: src/DOMupdates.js [hunk 1]
```diff
@@ -53,11 +43,11 @@ let domUpdates = {
     if (answer === true) {
       $("#js-show-answer").text('YOU GUESSED CORRECTLY!');
       $(`#${id}`).html('')
-      console.log(id)
       $('#js-guess-input').val('')
```
Comment (robbiejaeger) [inline src/DOMupdates.js]: Since this line is repeated in both conditions, you can bring it outside of the if/else to reduce the repetition.

File: src/Turn.js [hunk 2]
```diff
@@ -1,33 +1,29 @@
...
-  //inputting the answer and check the answer
-  //update score based on the answer
-  //if there is no wager, still check the answer and updare score
-  //round class determines what comes to the dashboard
-
 
 evaluateGuess(guess) {
   if(this.currentClue.answer === guess) {
     this.currentPlayer.addScore(this.currentClue.pointValue)
     this.round.continueRound()
```
Comment (robbiejaeger) [inline src/Turn.js]: Same comment here as above. Extract the lines that repeat within the if/else and put them outside of the if/else if they do not depend on a particular condition. That way the code is not repeated for both conditions.

File: src/index.js [hunk 3]
```diff
@@ -36,35 +31,31 @@ $('#js-names-button').click(function(e) {
...
-    e.preventDefault();
-    let guess = $('#js-guess-input').val();
-    let answer = game.round.currentTurn.evaluateGuess(guess)
-    console.log('answer', answer)
-    console.log(game.round.currentTurn.currentCard);
-    
-    domUpdates.giveFeedback(answer, game.round.currentTurn.currentCard);
-    game.round.changePlayer();
-  })
+$('#js-guess-button').click(function(e) {
```
Comment (robbiejaeger) [inline src/index.js]: Good to see this adding of an event listener was extracted so multiple event listeners weren't added to the same element for each click.

File: src/round.js [hunk 4]
```diff
@@ -18,10 +18,11 @@ class Round {
   }
 
   beginRound() {
-    this.currentTurn = new Turn(this.findPlayer(), this)
+    this.currentTurn = new Turn(this.players[0], this)
```
Comment (robbiejaeger) [inline src/round.js]: Interesting that you are passing the instance of the round in the instantiation of the Turn. Why is this necessary? Could you do without this?

**Comment by robbiejaeger** [comment]
Added a few comments in the PR.
